src/signals/incident-management/containers/Dashboard/components/Filter/styled.tsx

```diff
@@ -28,9 +32,14 @@ export const Select = styled.div<{
   display: flex;
   align-items: center;
   margin-right: ${themeSpacing(9)};
+  height: ${themeSpacing(14)};
+  margin-left: ${themeSpacing(12)};
 
-  &:first-of-type {
-    margin-left: ${themeSpacing(12)};
+  @media ${breakpoint('min-width', 'tabletM')} {
+    margin-left: unset;
+    &:first-of-type {
+      margin-left: ${themeSpacing(12)};
```

Comment:
According to desgin the left-margin should be 61px.

Reply:
will do!

Reply:
Ah, friday Linda told me to keep a margin of 40. For the components at least. Not sure if that also applies to the filter.